sys/include/usb/usbus.h

```diff
@@ -195,16 +208,17 @@ typedef struct {
         /**
          * @brief USBUS generic header generator generated length
          *
-         * Must return the length of the header that will be generated by
-         * @ref get_header
+         * Must return the length total length of the descriptors that will be
```

Comment:
```suggestion
         * Must return the total length of the descriptor that will be
```

Reply:
Same as below, multiple descriptors can be generated

Reply:
Clarified this a bit

Reply:
Alright